components/core/src/main/java/com/bloxbean/cardano/yaci/store/core/service/BlockFetchService.java

```diff
@@ -57,6 +59,10 @@ public class BlockFetchService implements BlockChainDataListener {
     private Integer previousEpoch;
     private Era previousEra;
 
+    @Setter
```

Comment:
@Sotatek-HuyLe3a Do we need setter also? or only getter is enough to avoid any change from outside?

Reply:
You're right, we need to remove Setter

Reply:
I updated @satran004